libraries/AP_HAL_ChibiOS/hwdef/CubePilot-CANMod/hwdef.dat

```diff
@@ -4,9 +4,13 @@
 # MCU class and specific type
 MCU STM32H7xx STM32H757xx
 
+define WATCHDOG_DISABLED
```

Comment:
Seems strange.  Likewise in the bootloader hwdef.

Reply:
Fixed